model_compiler/README.md

```diff
@@ -15,17 +15,40 @@ MIVisionX allows hundreds of different [OpenVX](https://www.khronos.org/registry
 <p align="center"><img width="100%" src="../docs/images/runtime.png" /></p>
 
 ## Pre-requisites
+
+### ONNX
 * MIVisionX libraries
```

Comment:
MIVisionX is common, just put the MIVisionX libraries first and link to https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX#build--install-mivisionx

Model Compiler is only supported in Linux, we have to put Ubuntu 16.04/18.04 or CentOS 7.5/7.6 as prerequisites